### PR TITLE
feat(support-panel): add subscription created date

### LINKED ISSUE #3981: Longer Duration subscription refunds
As a subscriber with a longer-duration subscription, so that I can change my mind about my purchase, I would like to be able to receive a full refund if I elect to in the first 30 days of my subscription

┆Issue is synchronized with this [Jira Story](https://jira.mozilla.com/browse/FXA-982)
┆Issue Number: FXA-982

Tasks:

- [x] Update Formatted Subscription Response [Jira Task: FXA-1334](https://jira.mozilla.com/browse/FXA-1334) - Pull Request: #4489
- [ ] Update Support Panel [Jira Task: FXA-1316](https://jira.mozilla.com/browse/FXA-1316) - Pull Request: #4502
- [X] Update Zendesk Widget [Jira Task: FXA-1317](https://jira.mozilla.com/browse/FXA-1317) - Pull Request: https://github.com/mozilla/omniscient/pull/11
- [x] Deploy Zendesk Widget [Jira Task: FXA-1318](https://jira.mozilla.com/browse/FXA-1318)

### PR DESCRIPTION
**Pull request by jackiemunroe**
- add the subscription creation timestamp to the information displayed

fixes #3981

~~Blocked by https://github.com/mozilla/fxa/pull/4489~~

### CODE REVIEW

File: packages/fxa-support-panel/lib/api.ts [hunk 1]
```diff
@@ -156,6 +157,7 @@ class SupportController {
 
     const formattedSubscriptions = subscriptions.map(s => ({
       ...s,
+      created: String(new Date(s.created * MS_IN_SEC)),
```
Comment (vbudhram) [inline packages/fxa-support-panel/lib/api.ts]: Are there tests that assert that the page elements exists?

File: packages/fxa-support-panel/test/lib/api.spec.ts [hunk 2]
```diff
@@ -76,6 +76,7 @@ function createDefaults(): MockCallsResponse {
     subscriptions: {
       response: [
         {
+          created: 1555354567,
```
Comment (chenba) [inline packages/fxa-support-panel/test/lib/api.spec.ts]: Could add a test to see if the template is rendered with the correct content.

File: packages/fxa-support-panel/lib/api.ts (hunk 1 above)
Comment (chenba) [reply packages/fxa-support-panel/lib/api.ts]: There are some existing tests for some elements, but not all of them.

Comment (jackiemunroe) [reply packages/fxa-support-panel/lib/api.ts]: > @jackiemunroe I wasn't able to test this locally, maybe something with my setup?
> 
> Screen Shot 2020-03-12 at 11 20 09 AM

Yeah, it was a bit of a journey to get it working locally for myself. I'm going to add a test to ensure the element exists

File: packages/fxa-support-panel/test/lib/api.spec.ts (hunk 2 above)
Comment (jackiemunroe) [reply packages/fxa-support-panel/test/lib/api.spec.ts]: I will do that. Thanks

File: packages/fxa-support-panel/lib/api.ts (hunk 1 above)
Comment (vbudhram) [reply packages/fxa-support-panel/lib/api.ts]: Ah ok, I'll defer on trying to debug my local setup for now :)

File: packages/fxa-support-panel/test/lib/api.spec.ts [hunk 3]
```diff
@@ -285,6 +286,22 @@ describe('Support Controller', () => {
...
+    const result = await server.inject({
+      headers: {
+        testing: 'example@example.com',
+      },
+      method: 'GET',
+      url: `/?uid=${uid}`,
+    });
+    cassert.equal(result.statusCode, 200);
+    const createdInfo = result.payload.match(
+      /<th>Created:<\/th>\s+<td>\w{3} \w{3} \d{1,2} \d{4} \d{1,2}:\d{2}:\d{2} GMT(-|\+)\d{4} \((\w\s?)+\)<\/td>/g
```
Comment (chenba) [inline packages/fxa-support-panel/test/lib/api.spec.ts]: Not messing around with that regex!

**Pull request merged**